src/test/frontend/deploy/deployfromsettings_controller_test.js

```diff
@@ -104,4 +108,53 @@ describe('DeployFromSettings controller', () => {
 
     expect(result).toEqual(expected);
   });
+
+  it('should not send empty description and container commands', () => {
```

Comment:
From my understanding the description states: unmodified fields should not be sent as empty strings "", but stay as null value. Took me a while to understand the difference, but ok